.eslintignore

```diff
@@ -1,3 +1,5 @@
 coverage
 dist
 node_modules
+fixtures
+packages/rule-webpack-config/docs/is-valid.md
```

Comment:
@sarvaje Can this be moved in `.eslintignore` file in `packages/rule-webpack-config/`?

Reply:
I'm going to try it

Reply:
We can't do that, we are using only this `.eslintignore`

Reply:
And if you create another `.eslintignore` in there it doesn't work?

Reply:
I think the only thing we can do is copy/paste this file to the folder `rule-webpack-config` and modify the package.json script to use that file instead of the one in the root folder.

Reply:
That's weird, usually ESLint uses the `.eslintrc` closest to the resource in combination with the one in the root so I would expect it to do the same with `.eslintignore`.